spec/mailers/user_mailer_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
```

Comment:
From what I read, a mailer spec just tests that the values brought to the mailer are rendered and that the mailer is delivered. This is one of the resources I used: https://www.lucascaton.com.br/2010/10/25/how-to-test-mailers-in-rails-with-rspec/